packages/@sanity/form-builder/src/sanity/DefaultAssetSource/utils/mimeType.ts

```diff
@@ -0,0 +1,132 @@
+interface MimeTypeInfo {
+  title: string
+}
+
+const MIME_TYPES: Record<string, MimeTypeInfo> = {
+  // Images
+  'image/bmp': {
+    title: 'Bitmap Image',
+  },
+  'image/jpeg': {
+    title: 'JPEG Image',
+  },
+  'image/gif': {
+    title: 'GIF Image',
+  },
+  'image/vnd.microsoft.icon': {
+    title: 'Icon',
+  },
+  'image/png': {
+    title: 'PNG Image',
+  },
+  'image/svg+xml': {
+    title: 'SVG Image',
+  },
+  'image/webp': {
+    title: 'WebP Image',
+  },
+  'image/tiff': {
+    title: 'TIFF Image',
+  },
+  'image/heic': {
+    title: 'HEIC Image',
+  },
+  // Audio
+  'audio/midi': {
+    title: 'MIDI Audio',
+  },
+  'audio/midi-x': {
+    title: 'MIDI Audio',
+  },
+  'audio/mpeg': {
+    title: 'MP3 Audio',
+  },
+  'audio/ogg': {
+    title: 'OGG Audio',
+  },
+  'audio/wav': {
+    title: 'WAW Audio',
```

Comment:
I think there's a typo here. Should be “WAV Audio”.

Reply:
Thanks, you are correct. Fixed in `next`.